doc/sailfish-webview-webengine.qdoc

```diff
@@ -0,0 +1,328 @@
+    engine context.
+
+    The \c{WebEngine} QML type is exposed through the \c{Sailfish.WebEngine} QML
+    import namespace.
+
+    Example of use:
+
+    \qml
+        import QtQuick 2.0
+        import Sailfish.Silica 1.0
+        import Sailfish.WebView 1.0
+        import Sailfish.WebEngine 1.0
+
+        ApplicationWindow {
+            id: root
+            initialPage: Component {
+                WebViewPage {
+                    id: page
+                    WebView {
+                        anchors.fill: parent
+                        active: true
+                        url: "http://www.sailfishos.org"
+                    }
+
+                    Component.onCompleted: {
+                        WebEngine.addObserver("clipboard:setdata")
+                        WebEngine.onRecvObserve.connect(function(message, data) {
+                            if (message == "clipboard:setdata" && !data.private) {
+                                console.log("Clipboard contents: " + data.data);
+                            }
+                        })
+                    }
+                }
+            }
+        }
+    \endqml
+
+    \sa WebView
+*/
+
+/*!
+    \readonly
+    \qmlproperty bool WebEngine::initialized
+    \brief Returns true if the component has been initialized, false otherwise.
+*/
+
+/*!
+    \qmlmethod WebEngine::addComponentManifest(manifestPath)
+    \brief Register JavaScript chrome components to be loaded into the WebEngine.
+
+    The sailfish WebEngine has three main layers. The top layer is exposed as
+    QML components with a developer-facing interface. The lowest level is the
+    native Gecko rendering engine.
+
+    Between the two sits a JavaScript chrome layer that can communicate with
+    either of the other layers using an observer/notification pattern. These
+    JavaScript components are considered privileged code and run with the
+    \l {https://developer.mozilla.org/en-US/docs/Mozilla/Gecko/Script_security#security_principals}{system principal}.
+
+    This call allows new JavaScript components to be added and executed as part
+    of this intermediary layer.
+
+    The \a manifestPath should be an absolute path pointing to a manifest file.
+    The manifest file should contain references to components in the same folder,
+    as shown in the following example. The example is comprised of three files:
+    a QML user interface file that calls \c addComponentManifest, the manifest
+    file and a component referenced in the manifest.
+
+    The WebEngine \b page.qml file calls \c addComponentManifest.
+
+    \qml
+        import QtQuick 2.1
+        import Sailfish.Silica 1.0
+        import Sailfish.Browser 1.0
+        import Sailfish.WebEngine 1.0
+
+        ApplicationWindow {
+            id: root
+            initialPage: Component {
+                WebViewPage {
+                    WebView {
+                        anchors.fill: parent
+                        active: true
```

Comment:
This is can be removed.

Reply:
Thanks! I've plucked it from here, and a couple of other places too.